### PR TITLE
Feature: Implement IconBrowser

### PR DESCRIPTION
**Pull request by Logianer**
**What?** The IconBrowser is now (half) complete.
I probably broke a few things along the way with the FA-imports because I don't have access to the Pro version of FA.

**Why?** I was bored and I like this project, so I pulled and botched something together.

Hope this helps in the developement of this "awesome" project

### CODE REVIEW

**Comment by MichaHuhn** [comment]
I guess, the Composition API is used in this project instead of the Options API.

**Comment by Logianer** [comment]
Yes this is debatable. You could rewrite everything into one or the other, but personally  I find the Options API more structured and readable. Probably because I don't know the Composition API...

**Comment by MichaHuhn** [comment]
Yeah, I like the Options API too, but @sebinside used the new Composition API in this project and when learning Vue, so I assume he's not familiar with the Options API.

**Comment by sebinside** [comment]
I know both APIs and I like the composition API way more as it is more honest (kinda blue pill / red pill, matrix style). Also, it is the default in the vue documentation, and although there are no plans to [deprecate the options API](https://vuejs.org/guide/extras/composition-api-faq.html#will-options-api-be-deprecated), I think, going with the default is always a good idea. Additionally, as this project already uses the composition API, I would opt for not mixing them (although it is technically feasible). 

TL;DR: I will look at this PR in the next coding stream and see what can be reused. If you want to use the chance to learn and play around with the composition API, feel free to do so ;)

**Comment by Logianer** [comment]
Let's not do any mixing of the APIs (in Germany we say "kein Mischmasch"). Rewrote my changes into the Composition API

**Review by sebinside — APPROVED** [review]
